.github/dependabot.yml

```diff
@@ -22,4 +24,11 @@ updates:
       - "dependency"
       - "gradle"
     commit-message:
-      prefix: "[No Ticket]"
+      prefix: "[WOR-1448]"
+    ignore:
+      - dependency-name: "org.springframework.boot:spring-boot-gradle-plugin"
+        update-types: [ "version-update:semver-major", "version-update:semver-minor", "version-update:semver-patch"]
+      - dependency-name: "com.google.apis:google-api-services-iam"
+        update-types: [ "version-update:semver-patch", "version-update:semver-minor" ]
+      - dependency-name: "com.google.apis:google-api-services-notebooks"
+        update-types: [ "version-update:semver-patch", "version-update:semver-minor" ]
```

Comment:
I'm sorry, I'm still not understanding why we want to let Dependabot create major version updates for any of these deps.  Please feel free to Slack me if it's easier to hash out there.

Reply:
The way the versioning works with these repos, I suspect that Dependabot would never realize the version was a major version update. The [specific case](https://github.com/DataBiosphere/terra-workspace-manager/pull/1611/commits/d5ee76a6357fe01ced60dc935f3fffd5f2a8bd2a) I added these for was lumped with the "minor release upgrades", but it was from v1->v2 with breaking package name changes.

Reply:
So yeah, I can see ignoring major version updates too for completeness, but I thought that these ignores would solve the specific pain I was feeling.

Reply:
Ticket for further investigation: https://broadworkbench.atlassian.net/browse/WOR-1446

Reply:
@okotsopoulos I simplified the configuration to just ignore all updates for those 3 libraries. Thanks for the suggestion!